lib/chewy/config.rb

```diff
@@ -18,6 +18,10 @@ class Config
                   # in tests with transactional fixtures or transactional
                   # DatabaseCleaner strategy.
                   #
+                  :console_strategy,
+                  # Allows to set default console strategy.
+                  # `:urgent` is used by default if `console_strategy` is not set
```

Comment:
Please move the doc above the setting. Lines 16-19 are about `use_after_commit_callbacks`